### ISSUE TITLE
Devices wwithout Serial Number

### ISSUE THREAD

**Issue by YoniTlz** [timrogers/litra#163]
Hi,

First of all thanks for this great library. I was searching for quite a while for something exactly like this so thank you 😊.
However, i have 2 Litra Glow connected to my computer but i can't seem to identify them as no serial number shows up in the device list.

I am running the scripts on Windows 10, with node 18.16.0.

Any idea why that is the case ?
<img width="659" alt="Capture d’écran 2023-05-29 000635" src="https://github.com/timrogers/litra/assets/22733661/926bf382-138d-4fb8-9a26-6f65efa995b2">

**Comment by timrogers** [comment]
I guess that there won't be anything we can do to get the serial numbers, and we'll have to add an option that lets you pick the device by its "index". But I'd like tot ry `hidapitester` first, just to get a sense of what's going on.

**Comment by YoniTlz** [comment]
Yeah, i've just started using index instead, works fine. I was just wondering if i you knew.
Here's the output :

`E:\Users\yonit\Documents\Stream\Touch Portal\scripts\litra>hidapitester --list-detail
0FD9/0078: Elgato - Elgato Facecam
  vendorId:      0x0FD9
  productId:     0x0078
  usagePage:     0xFF00
  usage:         0x0001
  serial_number: FW42K1A02217
  interface:     2
  path: \\?\HID#VID_0FD9&PID_0078&MI_02#8&252179c7&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C548: Logitech - USB Receiver
  vendorId:      0x046D
  productId:     0xC548
  usagePage:     0x0001
  usage:         0x0006
  serial_number:
  interface:     0
  path: \\?\HID#VID_046D&PID_C548&MI_00#8&31e90214&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}\KBD

046D/C548: Logitech - USB Receiver
  vendorId:      0x046D
  productId:     0xC548
  usagePage:     0x0001
  usage:         0x0002
  serial_number:
  interface:     1
  path: \\?\HID#VID_046D&PID_C548&MI_01&Col01#8&e11c452&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C900:  -
  vendorId:      0x046D
  productId:     0xC900
  usagePage:     0x000C
  usage:         0x0001
  serial_number:
  interface:     -1
  path: \\?\HID#VID_046D&PID_C900&Col01#8&15834a52&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C900:  -
  vendorId:      0x046D
  productId:     0xC900
  usagePage:     0x000C
  usage:         0x0001
  serial_number:
  interface:     -1
  path: \\?\HID#VID_046D&PID_C900&Col01#9&1318cabc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C900:  -
  vendorId:      0x046D
  productId:     0xC900
  usagePage:     0xFF43
  usage:         0x0202
  serial_number:
  interface:     -1
  path: \\?\HID#VID_046D&PID_C900&Col02#8&15834a52&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C900:  -
  vendorId:      0x046D
  productId:     0xC900
  usagePage:     0xFF43
  usage:         0x0202
  serial_number:
  interface:     -1
  path: \\?\HID#VID_046D&PID_C900&Col02#9&1318cabc&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C548: Logitech - USB Receiver
  vendorId:      0x046D
  productId:     0xC548
  usagePage:     0xFF00
  usage:         0x0001
  serial_number:
  interface:     2
  path: \\?\HID#VID_046D&PID_C548&MI_02&Col01#8&15c57970&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C548: Logitech - USB Receiver
  vendorId:      0x046D
  productId:     0xC548
  usagePage:     0xFF00
  usage:         0x0002
  serial_number:
  interface:     2
  path: \\?\HID#VID_046D&PID_C548&MI_02&Col02#8&15c57970&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C548: Logitech - USB Receiver
  vendorId:      0x046D
  productId:     0xC548
  usagePage:     0x000C
  usage:         0x0001
  serial_number:
  interface:     1
  path: \\?\HID#VID_046D&PID_C548&MI_01&Col02#8&e11c452&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}

046D/C548: Logitech - USB Receiver
  vendorId:      0x046D
  productId:     0xC548
  usagePage:     0x0001
  usage:         0x0080
  serial_number:
  interface:     1
  path: \\?\HID#VID_046D&PID_C548&MI_01&Col03#8&e11c452&0&0002#{4d1e55b2-f16f-11cf-88cb-001111000030}

0B05/1939: AsusTek Computer Inc. - AURA LED Controller
  vendorId:      0x0B05
  productId:     0x1939
  usagePage:     0xFF72
  usage:         0x00A1
  serial_number: 9876543210
  interface:     2
  path: \\?\HID#VID_0B05&PID_1939&MI_02#8&2765eff3&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}`

**Comment by timrogers** [comment]
That's helpful - thanks! I think I'll probably make a change to make `litra-devices` return an index and then to allow you to specify a device by its index.
